source/triggers/database-triggers.txt

```diff
@@ -137,81 +151,148 @@ Database Triggers have the following configuration options:
-          change events rather than **Update** events.
+       - :guilabel:`Cluster Name`. The name of the MongoDB cluster that the 
+           Trigger is associated with.
+      
+       - :guilabel:`Database Name`. The MongoDB database that contains the watched
+         collection.
 
-   * - :guilabel:`Full Document`
+       - :guilabel:`Collection Name`. The MongoDB collection to watch. Optional.
+         If you leave this option blank, the Source Type changes to "Database."
 
-     - If enabled, **Update** change events include the latest
-       :manual:`majority-committed </reference/read-concern-majority/>`
-       version of the modified document *after* the change was applied in
-       the ``fullDocument`` field.
+       - :guilabel:`Operation Type`. The collection operation types 
+         that cause the Trigger to fire. Select the operation types you want the 
+         trigger to respond to. Options include:
+       
+         - Insert
+         - Update 
+         - Replace
+         - Delete
 
-       .. note::
+         .. note::
+       
+           Update operations executed from MongoDB Compass or the MongoDB Atlas
+           Data Explorer fully replace the previous document. As a result,
+           update operations from these clients will generate **Replace**
+           change events rather than **Update** events.
+       
+       - :guilabel:`Full Document`. If enabled, **Update** change events include 
+         the latest :manual:`majority-committed </reference/read-concern-majority/>`
+         version of the modified document *after* the change was applied in
+         the ``fullDocument`` field.
+       
+         .. note::
+       
+            Regardless of this setting, **Insert** and **Replace** events always 
+            include the ``fullDocument`` field. **Delete** events never include 
+            the ``fullDocument`` field.
+       
+       - :guilabel:`Document Preimage`. Disabled for Collection and Database 
+         sources to limit unnecessary watches on the cluster for a new 
+         collection being created.
 
-          Regardless of this setting:
-          
-          - **Insert** and **Replace** events always include the
-            ``fullDocument`` field.
-          
-          - **Delete** events never include the ``fullDocument`` field.
+   * - :guilabel:`Database`
+     - | 
 
-   * - :guilabel:`Document Preimage`
+       - :guilabel:`Cluster Name`. The name of the MongoDB cluster that the 
+         Trigger is associated with.
+       
+       - :guilabel:`Database Name`. The MongoDB database to watch. Optional. 
+         If you leave this option blank, the Source Type changes to "Deployment," 
+         unless you are on a shared tier, in which case App Services will not 
+         let you save the trigger.
+
+       - :guilabel:`Operation Type`. The :ref:`database operation types 
+         <database-event-operation-types>` that cause the Trigger to fire. 
```

Comment:
[Q] Only this Database row has a link, do we want to have a link for `Collection` and `Deployment` as well?